projects/index.html

```diff
@@ -1,11 +1,11 @@
 ---
-title: Projects
-excerpt: "This is where we keep a collection of some of the projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
+title: Project Showcase
+excerpt: "This is where we keep a collection of some of the most interesting projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
```

Comment:
Hmmm... I see your point. I'm trying to come up with language that makes it clear that it's just a set of  projects that people have shown us and isn't examples. How do you think the sentence can be changed to include that? "this is where we keep a collection of user-submitted projects..."?